src/components/Table/Table.mdx

```diff
@@ -0,0 +1,30 @@
+import { Preview, Story, ArgsTable, PRIMARY_STORY } from '@storybook/addon-docs/blocks';
+import Table from './Table.vue';
+
+# Table
+Table is a flexible component that display sets of data
```

Comment:
Can we add a note here about what the spacing props are and what they do? And maybe a second sentence about how we envision devs styling the content inside `TableHeader` and `TableRow` with any other non-spacing styles?